src/constants/Typography.ts

```diff
@@ -14,17 +13,16 @@ const Text = styled.span<{
   lineHeight?: number;
   letterSpacing?: number;
 }>`
-  font-size: ${({ _fontSize }) => pxToRem(_fontSize)};
+  font-size: ${({ _fontSize }) => `${_fontSize}px`};
   text-align: ${({ textAlign }) => textAlign ?? "left"};
   font-weight: ${({ weight }) => weight ?? NORMAL_WEIGHT};
   font-family: ${({ _fontFamily }) => _fontFamily ?? "NotoSansKR"};
   color: ${({ _color }) => _color ?? Colors.black};
```

Comment:
```suggestion
   text-align: ${({ textAlign = "left" }) => textAlign};
  font-weight: ${({ weight = NORMAL_WEIGHT }) => weight};
  font-family: ${({ _fontFamily = "NotoSansKR" }) => _fontFamily};
  color: ${({ _color = Colors.black }) => _color};
```

Nullish coalescing operator말고 default 값 줘서 하는 게 어때요? 가독성이 더 좋아 보이지 않나요?

Reply:
훨씬 좋네요..! 대박 emotion 에서 Default valuet사용하는것 감사합니다!